xpk.py

```diff
@@ -1937,6 +1944,28 @@ def get_cluster_configmap(args, configmap_name):
   return config_map
 
 
+def create_vertex_tensorboard(args) -> map:
+  """Creates a Tensorboard instance in Vertex AI.
+
+  Args:
+    args: user provided arguments.
+
+  Returns:
+    map containing Tensorboard instance name, id and location.
+  """
+  tensorboard_config = {}
+  tensorboard_name = f'{args.cluster}-tb-instance' if args.tensorboard_name is None else args.tensorboard_name
+  instance_id = tensorboard.create_instance(project=args.project,  # pylint: disable=used-before-assignment
```

Comment:
are there errors from create_instance we should expose upwards to the user? Say they had bad configuration of the tensorboard

Reply:
The `cloud-accelerator-diagnostics` package will output those errors on the terminal.